From: Julian Ospald <hasufell@gentoo.org>
Date: Tue Aug 21 19:06:54 UTC 2012
Subject: desktop file

fix QA issues:

# desktop-file-validate /usr/share/applications/gargoyle.desktop
/usr/share/applications/gargoyle.desktop: error: file contains at least one line ending with a carriage return, while lines should only be separated by a line feed character. First such line is: "[Desktop Entry]"
/usr/share/applications/gargoyle.desktop: error: (will be fatal in the future): value "gargoyle-house.png" for key "Icon" in group "Desktop Entry" is an icon name with an extension, but there should be no extension as described in the Icon Theme Specification if the value is not an absolute path

--- garglk/gargoyle.desktop
+++ garglk/gargoyle.desktop
@@ -1,9 +1,9 @@
-[Desktop Entry]
-Version=1.0
-Type=Application
-Name=Gargoyle
-GenericName=Interactive Fiction interpreter
-Comment=Interactive Fiction multi-interpreter that supports all major IF formats
-Icon=gargoyle-house.png
-Exec=gargoyle
-Categories=Game;
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Gargoyle
+GenericName=Interactive Fiction interpreter
+Comment=Interactive Fiction multi-interpreter that supports all major IF formats
+Icon=gargoyle-house
+Exec=gargoyle
+Categories=Game;
